docs/src/main/sphinx/functions/json.rst

```diff
@@ -37,7 +37,7 @@ for keywords and identifiers.
 JSON path syntax and semantics
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A JSON path expression, similar to an SQL expression, is a recursive structure.
+JSON path expressions, similar to SQL expressions, are recursive structures.
```

Comment:
the similarity to SQL expressions doesn't do anything for the user IMO. Just saying they are recursive structures would be simpler IMO.